lib/signaling/v2/peerconnection.js

```diff
@@ -162,6 +167,9 @@ class PeerConnectionV2 extends StateMachine {
         writable: true,
         value: false
       },
+      _isRTCRtpSenderParamsSupported: {
```

Comment:
looks like no test is using this option to mock. we could get this out of options and use variable `isRTCRtpSenderParamsSupported` directly. But again, having a test that mocks this would be better.

Reply:
@makarandp0 Yup. We need to do this in order to be able to write unit tests.

Reply:
My suggestion was to either a) write a unit test that uses this mocking mechanism, or remove this for now, it can always be added when we need it to write such test. No point in keeping dead code around.